common/services/app/google-analytics.tsx

```diff
@@ -39,6 +39,8 @@ export const Ga4DataLayer: FunctionComponent<Props> = ({ data }) => {
               return toggle;
             case false:
               return `!${toggle}`;
+            default:
+              return null;
```

Comment:
@gestchild I know you look at making sure checking the value of toggles returned useful info if it was undefined, I just want to make sure that this change is OK with that?

Needed it because of the https://eslint.org/docs/latest/rules/array-callback-return rule